l1-contracts/hardhat.config.ts

```diff
@@ -13,18 +13,18 @@ if (!process.env.CHAIN_ETH_NETWORK) {
   require("dotenv").config();
 }
 
-const COMPILER_VERSION = "1.5.0";
-const PRE_RELEASE_VERSION = "prerelease-a167aa3-code4rena";
-function getZksolcUrl(): string {
-  // @ts-ignore
-  const platform = { darwin: "macosx", linux: "linux", win32: "windows" }[process.platform];
-  // @ts-ignore
-  const toolchain = { linux: "-musl", win32: "-gnu", darwin: "" }[process.platform];
-  const arch = process.arch === "x64" ? "amd64" : process.arch;
-  const ext = process.platform === "win32" ? ".exe" : "";
+// const COMPILER_VERSION = "1.5.3";
+// const PRE_RELEASE_VERSION = "prerelease-a167aa3-code4rena";
+// function getZksolcUrl(): string {
+//   // @ts-ignore
+//   const platform = { darwin: "macosx", linux: "linux", win32: "windows" }[process.platform];
+//   // @ts-ignore
```

Comment:
probably better to delete the line and supply an explicit compiler version in the config

Reply:
Outdated but not fully resovled